src/home/celebration/NftReward.tsx

```diff
@@ -0,0 +1,166 @@
+import { BottomSheetView } from '@gorhom/bottom-sheet'
+import { isPast, isToday } from 'date-fns'
+import differenceInDays from 'date-fns/differenceInDays'
+import React, { useEffect, useMemo, useRef } from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+import { HomeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { openDeepLink } from 'src/app/actions'
+import { BottomSheetRefType } from 'src/components/BottomSheet'
+import BottomSheetBase from 'src/components/BottomSheetBase'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import { nftRewardDisplayed } from 'src/home/actions'
+import { isSameNftContract } from 'src/home/celebration/utils'
+import { nftCelebrationSelector, showNftRewardSelector } from 'src/home/selectors'
+import i18n from 'src/i18n'
+import { nftsWithMetadataSelector } from 'src/nfts/selectors'
+import { useDispatch, useSelector } from 'src/redux/hooks'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { formatDistanceToNow } from 'src/utils/time'
+
+export default function NftRewardBottomSheet() {
+  const { t } = useTranslation()
+  const dispatch = useDispatch()
+
+  const canShowNftReward = useSelector(showNftRewardSelector)
```

Comment:
nit: maybe you don't need to check this again because it is checked in the WalletHome already?